### ISSUE TITLE
Syntax error: Unexpected token

### ISSUE THREAD

**Issue by misterunknown** [elasticsearch-dump/elasticsearch-dump#41]
Hi there,

I have problems dumping an index:

```
# ./elasticdump --input=http://127.0.0.1:9200/testindex/ --output=es_dump.json
Wed, 13 Aug 2014 13:44:49 GMT | starting dump

SyntaxError: Unexpected token '
    at Object.parse (native)
    at Request.requestResonse [as _callback] (/opt/elasticdump/lib/transports/elasticsearch.js:303:25)
    at Request.self.callback (/opt/elasticdump/node_modules/request/request.js:123:22)
    at Request.EventEmitter.emit (events.js:98:17)
    at Request.<anonymous> (/opt/elasticdump/node_modules/request/request.js:1047:14)
    at Request.EventEmitter.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (/opt/elasticdump/node_modules/request/request.js:998:12)
    at IncomingMessage.EventEmitter.emit (events.js:117:20)
    at _stream_readable.js:920:16
    at process._tickCallback (node.js:415:13)
```

What I don't understand is that all my other indexes can be dumped without problems. Only this one does not work. It is the only index with ~1M entries. The other indices are quite smaller ~100-1000 entries.
Any idea what I can do?

Cheers, Marco

**Comment by x0v** [comment]
@misterunknown I am facing the same error probably due to same reason. Just wanted to ask, how did you then finally dumped the data?

**Comment by misterunknown** [comment]
@x0v Well, that was a long time ago. As far as I remember, I dropped the index and reimported the data with the fixed import script. 

Meanwhile, we don't backup our ES indexes at all. In some (big) setups we have a multiple nodes which provide the data if one instance crashes, in other (small) setups we established a mechanism which can fill the index with data from the leading database (e.g. MySQL) again. This is usually not much slower than restoring a backup created with `es_backup_restore`, while the data are up-to-date. But the performance of a restore is bad either way.

If you really want to backup your indices, try to use [ES snapshots](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html). This is a mechanism implemented in ES since version 1.x, but wasn't really usable at that time (IMHO). But I evaluated this feature in version 5.0 and considered it suitable for backups. ES snapshots have one big advantage over other backup mechanisms I tried: the restore is blazing fast.

I hope I could help a bit ;)
